### ISSUE TITLE
.NET Core 3 is end of life on 13 December 2022

### ISSUE THREAD

**Issue by pctimhk** [loic-sharma/BaGet#792]
.NET Core 3 is end of life on 13 December 2022

**Comment by seriouz** [comment]
@pctimhk Yeah, if you want .NET 8 you can take a look on the [BaGetter fork](https://github.com/bagetter/BaGetter).
